examples/delete_server_by_id.py

```diff
@@ -0,0 +1,15 @@
+# delete_server_by_id.py
+from cbw_api_toolbox.cbw_api import CBWApi
+
+API_KEY = ''
+SECRET_KEY = ''
+API_URL = ''
+
+SERVER_ID = '' # For this example, you need to specify a server id in addition to your credentials
+
+result = CBWApi(API_URL, API_KEY, SECRET_KEY).delete_server_by_id(SERVER_ID)
+
+if result is True:
```

Comment:
I don't think it is useful to verify that True is True

Reply:
Done